core/src/test/java/org/apache/iceberg/rest/responses/TestLoadTableResponseParser.java

```diff
@@ -56,11 +59,14 @@ public void missingFields() {
         .hasMessage("Cannot parse missing field: metadata");
   }
 
-  @Test
-  public void roundTripSerde() {
+  private static int[] tableVersions = IntStream.range(2, 3).toArray();
+
+  @ParameterizedTest
+  @FieldSource("tableVersions")
```

Comment:
having `@ValueSource(ints = {2, 3})` is probably easier here than having to define a separate field for the input params

Reply:
It certainly is, less is more 👍